index.js

```diff
@@ -194,7 +205,8 @@ export function setGenericPassword(
     getOptionsArgument(serviceOrOptions),
     username,
     password,
-    getMinimumSecurityLevel(serviceOrOptions)
+    getMinimumSecurityLevel(serviceOrOptions),
+    getUseStrongBoxOption(serviceOrOptions)
```

Comment:
the number of parameters changed, did you verify this works on iOS (I'm afraid it won't)


I'd find it preferable for `setGenericPasswordForOptions` to accept an `options` object, instead of expanding the number of parameters is accepts (now 6).

can you please turn this

`public void setGenericPasswordForOptions(String service, String username, String password, String minimumSecurityLevel, boolean useStrongBox, Promise promise)`

into this?

`public void setGenericPasswordForOptions(String service, String username, ReadableMap options, Promise promise)`